mbzirc_ign/test/helper/TestFixture.hh

```diff
@@ -83,9 +83,16 @@ class MBZIRCTestFixture : public ::testing::Test
   /// \param[in] World file name.
   public: void LoadWorld(std::string _world)
   {
+    std::string foo;
+    ignition::common::env("IGN_GAZEBO_RESOURCE_PATH", foo);
+
+    std::cout << foo << std::endl;
```

Comment:
remove?

Reply:
Yep, leftover from cleaning up the environment hooks.